pandas/tests/io/pytables/test_compat.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+
+import pandas as pd
+
+from pandas.util.testing import assert_frame_equal
+
+
+tables = pytest.importorskip('tables')
+
+
+@pytest.fixture
+def pytables_hdf5_file(tmp_path):
+    """Use PyTables to create a simple HDF5 file.
+
+    There is no need for temporary file cleanup: pytest's `tmp_path` fixture
+    guarantees file system isolation between tests, retains files for later
+    inspection and then removes them in a rolling fashion so that the storage
+    space consumption does not grow indefinitely with the number of test runner
+    invocations.
+    """
+
+    table_schema = {
+        'c0': tables.Time64Col(pos=0),
+        'c1': tables.StringCol(5, pos=1),
+        'c2': tables.Int64Col(pos=2),
+    }
+
+    t0 = 1561105000.0
+
+    testsamples = [
+        {'c0': t0, 'c1': 'aaaaa', 'c2': 1},
+        {'c0': t0 + 1, 'c1': 'bbbbb', 'c2': 2},
+        {'c0': t0 + 2, 'c1': 'ccccc', 'c2': 10**5},
+        {'c0': t0 + 3, 'c1': 'ddddd', 'c2': 4294967295},
+    ]
+
+    objname = 'pandas_test_timeseries'
+
+    # The `tmp_path` fixture provides a temporary directory unique to the
```

Comment:
can you use ensure_clean_path; we don't use the pytest fixture at this time, though you can open an issue to change (many occurrences of this I suppose).